packages/atomic/cypress/integration/facets/date-facet.cypress.ts

```diff
@@ -74,8 +73,7 @@ function setupCustomDateFacet(
 describe('Date Facet with automatic ranges generated', () => {
   beforeEach(() => {
     setupAutoDateFacet(dateFacetProp.field, dateFacetProp.label);
-    createAliasShadow(dateFacetProp.field, FacetSelectors.dateFacet);
-    createAliasFacetUL(dateFacetProp.field, FacetSelectors.dateFacet);
+    createFacetAlias(dateFacetProp.field, FacetSelectors.dateFacet);
```

Comment:
Why were the test needed to be modified ?

I was under the impression that prod build is not "really" different from dev build, just more optimized.

Something I'm missing ?

Reply:
I refactored them a bit, since they were failing, now I'm checking if changing the browser to chrome will make them less flaky